.github/workflows/generator.yaml

```diff
@@ -19,32 +19,20 @@ jobs:
   generation:
     name: Generated artifacts
     runs-on: ubuntu-20.04
+    container: 
+      image: swiftnav/libsbp-build:latest-master
```

Comment:
Will need to retain libsbp-build image and update rather than using the master image all the time